worker/peergrouper/worker_test.go

```diff
@@ -645,41 +645,13 @@ func (s *workerSuite) TestMongoNoSpaces(c *gc.C) {
 		}
 
 		w := startWorkerSupportingSpaces(c, st, ipVersion)
-		runWorkerUntilMongoStateIs(c, st, w, state.MongoSpaceValid)
+		runWorkerUntilMongoStateIs(c, st, w, state.MongoSpaceInvalid)
 
-		// Only space one has all three servers in it
+		// No space was set.
 		c.Assert(st.getMongoSpaceName(), gc.Equals, "")
 	})
 }
 
-func (s *workerSuite) TestMongoSpaceNotOverwritten(c *gc.C) {
```

Comment:
Why get rid of this test? Isn't it still relevant in 2.3?

Reply:
AFAICT, it tests nothing.

The first part that auto-detects the "one" space (already in another test).

The second part sets the space state to _unknown_ in order to change the space to "testing" - no worker is  run; it just asserts the value is what was set.

The name doesn't match the test and the comment doesn't match the assertion. It looks like something that was never actually finished.